lib/stat_tracker.rb

```diff
@@ -0,0 +1,84 @@
+require "csv"
+
+class StatTracker
+
+  def self.from_csv(locations)
+    StatTracker.new(locations)
+  end
+
+  attr_reader :games, :teams, :game_teams
+
+  def initialize(locations)
+    @games = create_games(locations[:games])
+    @teams = create_teams(locations[:teams])
+    @game_teams = create_game_teams(locations[:game_teams])
+  end
+
+# helper methods
```

Comment:
Add context to the comment and indent it within the code block

```suggestion
  # Helper methods for initialization
```